admin/load.php

```diff
@@ -336,3 +336,88 @@ function perflab_get_module_data( $module_file ) {
 
 	return $module_data;
 }
+
+/**
+ * Initialise Admin Pointer
+ *
+ * Handles the bootstrapping of the admin pointer.
+ * Mainly jQuery code that is self-initialising.
+ *
+ * @since 1.0.0
+ * @return void
+ */
+function perflab_admin_pointer() {
+
+	wp_enqueue_style( 'wp-pointer' );
+	wp_enqueue_script( 'wp-pointer', false, array( 'jquery' ) );
+
+	$current_user = get_current_user_id();
+	$meta_key     = 'my-pointer-slug-dismissed';
```

Comment:
Can we update the meta key and pointer slug to something more specific?

```suggestion
	$meta_key     = 'perflab-intro-pointer-dismissed';
```

Reply:
Updated, thanks @jjgrainger